Makefile

```diff
@@ -10,10 +10,12 @@ VERSION ?= $(shell git describe --tags --exact-match 2>/dev/null || git rev-pars
 KS_VERSION ?= $(shell cd kumascript && git describe --tags --exact-match 2>/dev/null || git rev-parse --short HEAD)
 export KUMA_REVISION_HASH ?= $(shell git rev-parse HEAD)
 export KUMASCRIPT_REVISION_HASH ?= $(shell cd kumascript && git rev-parse HEAD)
+export KUMASCRIPT2_REVISION_HASH ?= $(shell cd kumascript2 && git rev-parse HEAD)
```

Comment:
I don't intend to leave this file as it is. The KUMASCRIPT2 variables are here as they are so that I could easily run the two versions side-by-side.

Before deploying, this file will be cleaned up.